BusinessCore.Tests/JVBBTests.cs

```diff
@@ -9,6 +12,89 @@ namespace BusinessCore.Tests
     [TestCategory("JVBB Tests")]
     public class JVBBTests
     {
+        public string VehiclesFilePath
+        {
+            get
+            {
+                return Path.Combine(Environment.CurrentDirectory, "Vehicles.xml");
+            }
+        }
+
+        public bool SameVehicle(Vehicle v1, Vehicle v2)
```

Comment:
Please, refactor this method using **fail faster** approach.
Make private, static and move it at the end of the class.

Reply:
Done, done, done and done :)